Intersect.Server.Core/Core/MapInstancing/Controllers/InstanceController.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Intersect.Server.Entities;
+
+namespace Intersect.Server.Core.MapInstancing.Controllers;
+public class InstanceController
+{
+    Guid InstanceId { get; set; }
+
+    public HashSet<Player> Players { get; set; } = new HashSet<Player>();
```

Comment:
New syntax -- just `new()` here.

Reply:
right, gonna take some getting used to. Love the new destructuring stuff in foreaches though, fixed